src/main/resources/templates/c/gcc/test/testUtils/AbstractTest.py

```diff
@@ -90,7 +92,7 @@ def start(self, testResults: Dict[str, Result], suite: TestSuite):
                 except TimeoutError:
                     self._timeout()
                 except Exception as e:
-                    self.__markAsFailed(f"'{self.name}' had an internal error. {str(e)}.\nPlease report this on Moodle (Detailfragen zu Programmieraufgaben)!")
+                    self.__markAsFailed(f"'{self.name}' had an internal error. {str(e)}.\nPlease report this on Zulip!")
```

Comment:
I think a more general message would be better here. (same below)
```suggestion
                    self.__markAsFailed(f"'{self.name}' had an internal error. {str(e)}.\nPlease report this to an instructor!")
```